src/main/java/org/opencraft/server/cmd/impl/CreeperCommand.java

```diff
@@ -57,21 +60,27 @@ public static CreeperCommand getCommand() {
   }
 
   public void execute(Player player, CommandParameters params) {
-    if (GameSettings.getBoolean("Tournament")) {
-      player.getActionSender().sendChatMessage("- &e/cr is disabled during the tournament.");
-      return;
-    }
-    Position pos = player.getPosition();
-    int px = (pos.getX() - 16) / 32;
-    int py = (pos.getY() - 16) / 32;
-    int pz = ((pos.getZ() - 16) / 32);
+    Thread creeperThread;
+    creeperThread = new Thread(
+      () -> {
+        player.creeperTime = System.currentTimeMillis();
 
-    player.creeperTime = System.currentTimeMillis();
+        World w = World.getWorld();
+        w.broadcast("- " + player.parseName() + " &eisn't feeling so good...");
+        w.broadcast("- &esssssssSSSSSSSSS");
 
-    World.getWorld().broadcast("- " + player.parseName() + " &eisn't feeling so good...");
-    World.getWorld().broadcast("- &esssssssSSSSSSSSS");
-    ((CTFGameMode)World.getWorld()
-        .getGameMode())
-        .explodeTNT(player, World.getWorld().getLevel(), px, py, pz, 4, true, true, false, null);
+        try {
+          Thread.sleep((long)(1000 * GameSettings.getFloat("CreeperTime")));
```

Comment:
If `CreeperTime < 0` Unhanded IllegalArgumentException gets thrown Maybe revert to old behavior and don't start thread if it's <= 0?

Reply:
Changed so now it uses 0 if it's negative. Same for CreeperRadius.